Source/nthread.h

```diff
@@ -23,5 +23,10 @@ void nthread_start(bool set_turn_upper_bit);
 void nthread_cleanup();
 void nthread_ignore_mutex(bool bStart);
 bool nthread_has_500ms_passed();
+/**
+ * @brief Calculates the progress in time to the next game tick
+ * @return progress in percent (0.0f to 1.0f)

```

Comment:
0-1 is a fraction, no %
```suggestion
 * @return Progress as a fraction (0.0f to 1.0f)
```